k8s/fields.py

```diff
@@ -151,3 +152,22 @@ class RequiredField(Field):
     def is_valid(self, instance):
         value = self.__get__(instance)
         return value is not None and super(RequiredField, self).is_valid(instance)
+
+
+class AnyField(Field):
```

Comment:
AnyField is not suposed to contain lists or dicts of class instances, only built-in types that are (de)serializable out of the box. We could allow class instances in there, but then you'd get the weirdness of the class instances turning into dicts on `Model.save`. The alternative is to add some checks to prevent class instances from being used.